<launch>

  <node name="add_objects_service" pkg="scene_objects" type="add_objects_service.py" respawn="false" output="screen">
  </node>

</launch>
